content/source/docs/cloud/api/team-access.html.md

```diff
@@ -22,6 +22,8 @@ page_title: "Team Acccess - API Docs - Terraform Cloud"
 
 # Team access API
 
+-> **Note:** Team management and access controls are a paid feature, available as part of the **Team** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing/).
```

Comment:
> Team management and access controls are a paid feature
Would suggest either "Team Management is a paid feature" or "Team Management and Role Based Access Control are paid features" (then again we may not be wanting to use the word "RBAC" for what we're offering right now. @glenngillen ?)

Reply:
> Would suggest either "Team Management is a paid feature"
👍

Reply:
Capitalizing "Team Management" elevates it to a pseudo-product-name status. Is that desired, here, or is it still just a feature description?